modules/ding_webtrekk/ding_webtrekk.module

```diff
@@ -80,8 +80,7 @@ function ding_webtrekk_panels_pane_content_alter($content, $pane, $args, $contex
  * Track internal search results.
  */
 function ding_webtrekk_views_post_execute(&$view) {
-  $count = $view->total_rows;
-  if ($view->name == 'ding_multiple_search' && $count > 0) {
+  if ($view->name == 'ding_multiple_search' && strpos(current_path(), 'search/node') === 0) {
```

Comment:
Why do we need the current path check here?

Reply:
Because else it's added on ting search page also because of https://platform.dandigbib.org/issues/629

Reply:
OK. Then I would appreciate a comment explaining this.

Reply:
Oh yeah sorry, of course. Coming up :)